src/android/Text.tsx

```diff
@@ -52,6 +53,10 @@ export class Text extends CommonText {
     focus() {
         AccessibilityUtil.setAccessibilityFocus(this);
     }
+
+    static setDefaultMaxFontSizeMultiplier(maxFontSizeMultiplier: number): void {
```

Comment:
This shouldn't be exposed as a static method on Text. It should be a method on the UserInterface namespace.